request_handlers.cc

```diff
@@ -48,10 +51,62 @@ void StaticRequestHandler::handleRequest(const char* request_buffer,
     std::cout << t << "." << std::endl;
   }
 
+  int i = 0;
+  auto curToken = tokens.begin();
+  std::string resourcePath;
+  for (;;) {
+    if (curToken == tokens.end()) {
+      // TODO: tokens is too short
+      std::cout << "DEBUG: tokens in RequestRouter is too short" << std::endl;
+      return;
+    }
+
+    if (i == 1) {
```

Comment:
Can we add a comment explaining that we expect the `resourcePath` to be the 2nd token, [as  seen here](https://github.com/UCLA-CS130/Mr.-Robot-et-al./blob/d2b02baa8f698c17a31ec5d4c8ffa8621fbecc91/request_router.cc#L22). 

Of course, this all temporary until we move HTTP request parsing into its own module. Actually, can you file an issue for that?